subsys/net/conn_mgr/conn_mgr_connectivity.c

```diff
@@ -263,6 +275,61 @@ static void conn_mgr_conn_handle_iface_admin_up(struct net_if *iface)
 	}
 }
 
+/**
+ * @brief Take the provided iface admin-down.
+ *
+ * Called automatically by conn_mgr when an iface loses connection and does not expect to regain it,
```

Comment:
What does `not expected to regain it` mean here, how do we know this? Or according to code below we just take the interface down according to user wishes (the flag is set). The text is slightly confusing as one gets impression that there is some heuristics to determine whether to take the interface down or not.

Reply:
You're right -- How about

`Called automatically by conn_mgr when an iface has lost connection and will not attempt to regain it`